apps/clock/manifest.webapp

```diff
@@ -15,6 +15,12 @@
     "audio-channel-alarm":{},
     "moz-extremely-unstable-and-will-change-webcomponents":{}
   },
+  "datastores-owned": {
+    "alarms": {
+      "access" : "readwrite",
```

Comment:
can we make this readonly? It seems like Lockscreen doesn't need to be able to edit the data.